spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerQueryOptions.java

```diff
@@ -67,13 +68,23 @@ public SpannerQueryOptions setIncludeProperties(Set<String> includeProperties) {
 		return this;
 	}
 
-	public Timestamp getTimestamp() {
```

Comment:
Seems like a breaking change. Can we just keep both instance variables?
It could be a bit confusing that you set an exact timestamp but the bound is set.

Reply:
added it back. The timestamp is inside the timestampBound too, so I am just extracting it from that.